hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconAndAdminContainerCLI.java

```diff
@@ -261,7 +261,8 @@ void testNodesInDecommissionOrMaintenance(
     // a new replica-copy is made to another node.
     // For maintenance, there is no replica-copy in this case.
     if (!isMaintenance) {
-      TestHelper.waitForReplicaCount(containerIdR3, 4, cluster);
+      GenericTestUtils.waitFor(() -> TestHelper.countReplicas(containerIdR3, cluster) == 4,
+          200, 30000);
```

Comment:
> Observed the condition was true for a very smaller span

The same problem may affect other tests (currently there is only one other usage).  I think the new, reduced check interval can be set in `waitForReplicaCount`.

Reply:
That is why I cross checked the uses of waitForReplicaCount which I found in TestContainerReplication, And I tried to see its flakiness..
- Worked 100% success for 10x10 https://github.com/raju-balpande/apache_ozone/actions/runs/8710014704 
- Worked 99.5% success for 20x20 https://github.com/raju-balpande/apache_ozone/actions/runs/8720977974
And seems no change require for this class.
Please suggest if I shall make this change in TestHelper.waitForReplicaCount instead.

Reply:
> no change require for this class

If it's not getting worse due to the change, please make it in `waitForReplicaCount`.

Reply:
I agree to your point. Even though the TestContainerReplication is not breaking now but the following two lines are indicating the temporary condition and hence thought to extend the change for this class as well. Hence making change in TestHelper.waitForReplicaCount only.
<img width="783" alt="image" src="https://github.com/apache/ozone/assets/146973984/91753787-4dfd-436b-a570-61b28af16844">